### ISSUE TITLE
1.3.2 Meaningful Sequence が「意味のある順序」でよいのか

### ISSUE THREAD

**Issue by bakera** [waic/wcag21#1535]
達成基準 1.3.2 Meaningful Sequence は「意味のある順序」と訳されています。

https://waic.jp/docs/WCAG21/Understanding/meaningful-sequence.html

しかし、この達成基準の達成方法や失敗例には、「順序」の問題ではないものも含まれています。具体的には:

- 単語内の文字間隔を制御するために、CSS の letter-spacing を使用する https://waic.jp/docs/WCAG21/Techniques/css/C8
- 達成基準 1.3.2 の失敗例 － 単語内の間隔を制御するために、空白文字を使用している https://waic.jp/docs/WCAG21/Techniques/failures/F34

これは、テキストの順序自体は正しいものの、間にスペースがあるため正しく読まれないという問題です。

原文では Sequence で「一連のもの」というような意味があるため、間にスペースが含まれるとMeaningful Sequenceにならないということは理解できます。しかし、日本語で「意味のある順序」と言われると「順序は正しいのでは?」と思えてしまいます。Order ではなく Sequence であるというニュアンスが出せていないように思います。

とはいえ妙案はないのですが……

**Comment by momdo** [comment]
sequenceを順序としないならば、連続ですかね…そうすると今度は順序はどこに行った？となりそうではあります。

**Comment by caztcha** [comment]
「意味をなす連なり」とか。

**Comment by bakera** [comment]
https://github.com/waic/wcag21/pull/1541 をみていたら " image sequence" を今は「画像の連続」としていて、今回のPRで「映像シーケンス」にしようとしていますね。
「連続」か「シーケンス」か。

**Comment by bakera** [comment]
「意味をなす連なり」が有力か。他の案としては「意味をなす流れ」「意味をなす一続き」。
「意味をなす並び」はorder感が強いか。G57には「意味のある順序で並べる」という表現があり、厳しそう。
「シーケンス」は映像について言うのはわかるが、そうでない文脈だと一般的ではないかもしれない。

「順序」は他の箇所にもたくさん出てくるので全体に影響がある。
一度全体を置き換えてみて、全体の影響はわからないかも。

**Comment by momdo** [comment]
「順番」とするのはどうか？

**Comment by bakera** [comment]
「並び」ではどうか?

**Comment by momdo** [comment]
用語集「正しく読む順序」にあわせて、「正しく読むシーケンス」としてみる。
